modules/standard/Math.chpl

```diff
@@ -545,16 +545,30 @@ module Math {
     return chpl_expm1(x);
   }
 
+  /* Returns the value of the argument `x` multiplied by 2 raised to the
+     argument `exp` power, i.e., ``x * 2**exp``. */
+  inline proc ldExp(x:real(64), exp:int(32)):real(64) {
+    return chpl_ldexp(x, exp);
+  }
+
+  /* Returns the value of the argument `x` multiplied by 2 raised to the
+     argument `exp` power, i.e., ``x * 2**exp``. */
+  inline proc ldExp(x:real(32), exp:int(32)):real(32) {
+    return chpl_ldexp(x, exp);
+  }
+
   /* Returns the value of the argument `x` multiplied by 2 raised to the
      argument `n` power, i.e., ``x * 2**n``. */
+  @deprecated(notes="'ldexp' with an 'n' argument has been deprecated, please use :proc:`ldExp` with an 'exp' argument instead")
```

Comment:
This was recently moved from AutoMath right? It might be nice to put a note in that deprecation that the name has changed